### PR TITLE
Added Flutter Scripts

### PR DESCRIPTION
**Pull request by KABILAN235**
## Description

Added Flutter Scripts that can create flutter apps from Raycast.

## Type of change
- [x] New script command
## Screenshot

Flutter
## Dependencies / Requirements

This Script Requires Flutter To Be Installed locally.
2 Variables Should be Configured to Use this script
SDK_PATH : Location of the Flutter SDK ( Will remove after we get support for path variables)
PROJECT_PATH: Location where your flutter projects are to be created
Tip 💡: Use $USER in the Place of Your Username :)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by nhojb** [comment]
Looks good. Thanks for contributing.

File: commands/developer-utils/flutter-create.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Flutter Create Project
+# @raycast.mode fullOutput
+
+# Optional parameters:
+# @raycast.icon images/flutter.png
+# @raycast.argument1 { "type": "text", "placeholder": "ProjectName" }
+# @raycast.packageName flutter-create
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:11]:
```suggestion
# @raycast.packageName Flutter
```

File: commands/developer-utils/flutter-create.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Flutter Create Project
+# @raycast.mode fullOutput
+
+# Optional parameters:
+# @raycast.icon images/flutter.png
+# @raycast.argument1 { "type": "text", "placeholder": "ProjectName" }
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:10]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Project Name" }
```

File: commands/developer-utils/flutter-create.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Flutter Create Project
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:5]:
```suggestion
# @raycast.title Create Project
```

File: commands/developer-utils/flutter-create.sh [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.mode fullOutput
+
+# Optional parameters:
+# @raycast.icon images/flutter.png
+# @raycast.argument1 { "type": "text", "placeholder": "ProjectName" }
+# @raycast.packageName flutter-create
+
+# Documentation:
+# @raycast.description Creates A Flutter Project from Raycast
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:14]:
```suggestion
# @raycast.description Creates a Flutter Project
```

File: commands/developer-utils/flutter-create.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+# @raycast.author Kabilan VS
+# @raycast.authorURL https://github.com/KABILAN235
+
+# Flutter SDK Path. Eg: /Users/$USER/Devtools/flutter/bin
+SDK_PATH="/Users/$USER/DevTools/flutter/bin"
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:19]: Is this a default path for Flutter? If not, we need to ask the user to set a its own path converting this Script Command into a template one. 

How to convert this Script Command into a template?
Rename the file from `flutter-create.sh` to `flutter-create.template.sh`

Then, add a comment after the SDK_PATH explaining the user must the PATH for the flutter bin folder.

File: commands/developer-utils/flutter-create.sh [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+# Default Project Path, ie- This is Where Your Projects will be Stored
+PROJECT_PATH="/Users/$USER/Desktop/Dev/FlutterProjects/"
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:22]: You need to ask the user to set its own workspace folder here, following basically the same approach I've mentioned on the previous comment.

File: commands/developer-utils/flutter-create.sh [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+export PATH="$PATH:$SDK_PATH"
+export PATH
```
Comment (unnamedd) [inline commands/developer-utils/flutter-create.sh:25]: You don't need this line here, you've already exported $PATH in the line 24.

**Review by unnamedd — COMMENTED** [review]
Hello @KABILAN235, thanks for your contribution, however it is not conformant to some rules we have for contributions in this repository. Even though it was already merged, I need you to do some changes in your Script Command and then submit them in a new PR. Please address the comments and let us know.

If you need any help, please let me know.

File: commands/developer-utils/flutter-create.sh (hunk 5 above)
Comment (KABILAN235) [reply commands/developer-utils/flutter-create.sh:19]: Yeah Sure, I'l Create A New PR With All Changes

Comment (unnamedd) [reply commands/developer-utils/flutter-create.sh:19]: Thanks @KABILAN235, as soon you create, please mention my user for a review! ;)

**Pull request merged**